--- vernemq.conf.192.orig	2019-09-10 13:07:59.000000000 +0000
+++ vernemq.conf	2021-05-03 05:18:11.745722205 +0000
@@ -7,7 +7,7 @@
 ## 
 ## Acceptable values:
 ##   - on or off
-allow_anonymous = off
+allow_anonymous = on
 
 ## Allow new client connections even when a VerneMQ cluster is inconsistent.
 ## 
@@ -92,7 +92,7 @@
 ## 
 ## Acceptable values:
 ##   - text
-## persistent_client_expiration = 1w
+persistent_client_expiration = 1m
 
 ## The maximum delay for a last will message. This setting
 ## applies only to MQTTv5 sessions and overrides the value provided by
@@ -140,7 +140,7 @@
 ## 
 ## Acceptable values:
 ##   - an integer
-max_offline_messages = 1000
+max_offline_messages = 10000
 
 ## This option sets the maximum MQTT size that VerneMQ will
 ## allow.  Messages that exceed this size will not be accepted by
@@ -152,7 +152,7 @@
 ## 
 ## Acceptable values:
 ##   - an integer
-max_message_size = 0
+max_message_size = 8192
 
 ## If a message is published with a QoS lower than the QoS of the subscription it is
 ## delivered to, VerneMQ can upgrade the outgoing QoS. This is a non-standard option.
@@ -233,7 +233,7 @@
 ## 
 ## Acceptable values:
 ##   - an IP/port pair, e.g. 127.0.0.1:10011
-listener.tcp.default = 127.0.0.1:1883
+listener.tcp.default = 0.0.0.0:1883
 
 ## 'listener.tcp.allowed_protocol_versions' configures which
 ## protocol versions are allowed for an MQTT listener. The allowed
@@ -254,7 +254,7 @@
 ## 
 ## Acceptable values:
 ##   - text
-## listener.tcp.allowed_protocol_versions = 3,4
+listener.tcp.allowed_protocol_versions = 3,4,5
 
 ## listener.vmq.clustering is the IP address and TCP port that
 ## the broker will bind to accept connections from other cluster
@@ -641,7 +641,7 @@
 ## 
 ## Acceptable values:
 ##   - an integer
-vmq_acl.acl_reload_interval = 10
+vmq_acl.acl_reload_interval = 60
 
 ## Set the path to a password file.
 ## 
@@ -658,7 +658,7 @@
 ## 
 ## Acceptable values:
 ##   - an integer
-vmq_passwd.password_reload_interval = 10
+vmq_passwd.password_reload_interval = 60
 
 ## Configure the vmq_diversity plugin script dir. The script dir
 ## is searched for Lua scripts which are automatically loaded when the
@@ -1334,7 +1334,7 @@
 ## 
 ## Acceptable values:
 ##   - text
-nodename = VerneMQ@127.0.0.1
+nodename = VerneMQ-01
 
 ## Cookie for distributed node communication.  All nodes in the
 ## same cluster should use the same cookie or they will not be able to
@@ -1345,7 +1345,7 @@
 ## 
 ## Acceptable values:
 ##   - text
-distributed_cookie = vmq
+distributed_cookie = dmagWHKuFHPC7ipp
 
 ## Sets the number of threads in async thread pool, valid range
 ## is 0-1024. If thread support is available, the default is 64.
@@ -1418,5 +1418,15 @@
 ## 
 ## Acceptable values:
 ##   - an integer
-leveldb.maximum_memory.percent = 70
+#leveldb.maximum_memory.percent = 70
+leveldb.maximum_memory = 104857600
+leveldb.write_buffer_size_min = 31457280
+leveldb.write_buffer_size_max = 62914560
+
+# Cluster inter-comm configuration
+erlang.distribution.port_range.minimum = 6000
+erlang.distribution.port_range.maximum = 7999
+
+topic_alias_max_client = 255
+topic_alias_max_broker = 255
